Runtime/API.cs

```diff
@@ -10,14 +10,43 @@ namespace StringSDK
 {
     public static class Constants
     {
-        public const string api_base_url = "http://localhost:5555";
+        public const string PROD_API_URL = "https://api.string-api.xyz";
+        public const string SANDBOX_API_URL = "https://api.sandbox.string-api.xyz";
+        public const string DEV_API_URL = "https://string-api.dev.string-api.xyz";
+        public const string LOCAL_API_URL = "http://localhost:5555";
     }
 
     public static class StringXYZ
     {
         // API Client
         static ApiClient apiClient;
 
+        public static EnvironmentType Environment
```

Comment:
Let's make this work the same way as the other static variables.  Also, consider moving the logic into its own function to keep the inline accessors/mutators concise.